common/reviews/api/rush-lib.api.ts

```diff
@@ -125,6 +125,14 @@ class LockStepVersionPolicy extends VersionPolicy {
   readonly version: string;
 }
 
+// @public
+class PnpmOptionsConfiguration {
+  // WARNING: The type "IPnpmOptionsJson" needs to be exported by the package (e.g. added to index.ts)
+  // @internal
```

Comment:
>internal [](start = 6, length = 8)

@private ?

Reply:
Good idea.  But this is an API Extractor feature request that's out of scope for this PR.

---
In reply to: [216065927](https://github.com/Microsoft/web-build-tools/pull/817#discussion_r216065927) [](ancestors = 216065927)